cpp/src/io/orc/writer_impl.cu

```diff
@@ -1975,7 +2159,11 @@ void writer::impl::write(table_view const& table)
 
     ProtobufWriter pbw_(&buffer_);
 
-    auto const statistics = gather_statistic_blobs(stats_freq_, orc_table, segmentation);
+    auto intermediate_stats = gather_statistic_blobs(stats_freq_, orc_table, segmentation);
+
+    if (intermediate_stats.stripe_stat_chunks.size() > 0)
```

Comment:
Nitpick: Please consider braces for this multi-line if-condition.

Reply:
❤️